### PR TITLE
Add/tests

### PR DESCRIPTION
**Pull request by mikebarr24**
Created tests and script for running tests.

### CODE REVIEW

**Comment by mikebarr24** [comment]
fixes #39

File: e2e/test_homepage.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import re
+from playwright.sync_api import Page, expect
+
+
+def test_homepage(page: Page):
+    page.goto("http://localhost:8000/")
+    # Expect a title "to contain" a substring.
+    expect(page).to_have_title(re.compile("Piccolo Theme"))
+
+    # Github Link
```
Comment (dantownsend) [inline e2e/test_homepage.py:10]: This should probably say `# Sidebar`

File: requirements/test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==2.0.0
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.5
+text-unidecode==1.3
+tomli==2.0.1
+tornado==6.2
+urllib3==1.26.12
```
Comment (dantownsend) [inline requirements/test-requirements.txt:43]: I'd be tempted to simplify this list a bit. I assume you did `pip freeze`, which is OK, but we should be able to get away with just these:
```playwright==1.27.1
pytest==7.1.3
pytest-playwright==0.3.0
Sphinx==5.2.3
sphinx-autobuild==2021.3.14
```

**Comment by dantownsend** [comment]
@mikebarr24 This is great, thanks! Just some minor comments.

File: requirements/test-requirements.txt (hunk 2 above)
Comment (mikebarr24) [reply requirements/test-requirements.txt]: Yea. This is what I did. I wasn't sure whether this was right or not. Cheers for the feedback!

**Comment by mikebarr24** [comment]
That should be the requested changes made to the PR. Let me know if that's grand and thanks for the feedback. 
I'll get a look at the next part this coming week.

**Comment by dantownsend** [comment]
@mikebarr24 This looks great - thanks for making those changes.

**Pull request merged**